lib/OpenQA/WebSockets/Controller/Worker.pm

```diff
@@ -120,7 +121,7 @@ sub _message {
         my $wid                   = $worker->{id};
 
         $worker_status->{$wid} = $json;
-        log_debug(sprintf('Received from worker "%u" worker_status message "%s"', $wid, Dumper($json)));
+        log_debug(sprintf('Received from worker "%u" worker_status message "%s"', $wid, YAML::XS::Dump($json)));
```

Comment:
Unrelated change?

Reply:
Ah yes, kind of.
I was debugging and comparing the logfiles with `vimdiff`. Data::Dumper output is by default unsorted and not well suited for a diff, IMHO.
I can take it out if you want.

Reply:
True, that's one of the reasons [Mojo::Util::dumper](https://github.com/mojolicious/mojo/blob/93606e4d453bb29fddae2ad296017ad9b50d8531/lib/Mojo/Util.pm#L141) exists. I have no preference, others might.

Reply:
I've done the same kind of temporary change before. I think it's excellent for debugging.

Reply:
I repeated the benchmark with a bigger hash and now times are almost equal:
```
perl -wE'
use YAML::XS ();
use Data::Dumper;
use Benchmark;
my $hash = YAML::XS::LoadFile("worker-status.yaml");
timethese($ARGV[0] || -1, {
  yaml => sub { YAML::XS::Dump($hash) },
  dumper => sub { Data::Dumper::Dumper($hash) },
});' 50000
Benchmark: timing 50000 iterations of dumper, yaml...
    dumper:  3 wallclock secs ( 2.44 usr +  0.00 sys =  2.44 CPU) @ 20491.80/s (n=50000)
      yaml:  2 wallclock secs ( 2.42 usr +  0.00 sys =  2.42 CPU) @ 20661.16/s (n=50000)
```
`worker-status.yaml`:
```yaml
---                                                                         
current_webui_host: http://localhost:47771
job:
  URL: http://localhost:20013/G9bUHr5eIMESFMlw
  assets:
    iso:
    - Core-7.2.iso
  assigned_worker_id: 1
  blocked_by_id: ~
  clone_id: ~
  group_id: ~
  id: 1
  name: tinycore-1-flavor-i386-Build1-core@coolone
  priority: 50
  result: none
  settings:
    ARCH: i386
    BUILD: '1'
    CDMODEL: ide-cd
    DISTRI: tinycore
    FLAVOR: flavor
    HDDMODEL: ide-drive
    INTEGRATION_TESTS: '1'
    ISO: Core-7.2.iso
    JOBTOKEN: G9bUHr5eIMESFMlw
    LOG_LEVEL: debug
    MACHINE: coolone
    NAME: 00000001-tinycore-1-flavor-i386-Build1-core@coolone
    OPENQA_HOSTNAME: localhost:47771
    PUBLISH_HDD_1: core-hdd.qcow2
    QEMU: i386
    QEMUPORT: 20012
    QEMU_NO_FDC_SET: '1'
    QEMU_NO_KVM: '1'
    QEMU_NO_TABLET: '1'
    RETRY_DELAY: 5
    RETRY_DELAY_IF_WEBUI_BUSY: 60
    TEST: core
    TESTING_ASSERT_SCREEN_TIMEOUT: '1'
    VERSION: '1'
    VNC: '91'
    WORKER_CLASS: qemu_i386,qemu_x86_64
    WORKER_HOSTNAME: 127.0.0.1
  state: scheduled
  t_finished: ~
  t_started: ~
  test: core
status: working
```